introduction/site_survey.md

```diff
@@ -0,0 +1,353 @@
+
+For each management node, the component name (xname), role, and subrole can be extracted from the SHCD. However, the rest of the
+MAC address information needs to be collected another way. Collect as much information as possible
+before the PIT node is booted from the LiveCD and then get the rest later when directed. See the scenarios
+which enable partial data collection below in [First Time Install](../install/prepare_configuration_payload#first_time_install).
+
+See [Create NCN Metadata CSV](../install/create_ncn_metadata_csv.md) for instructions about creating this file.
+
+<a name="switch_metadata_csv"></a>
+### `switch_metadata.csv`
+
+The `switch_metadata.csv` file is manually created to include information about all spine, aggregation, CDU,
+and leaf switches in the system. None of the Slingshot switches for the HSN should be included in this file.
+
+See [Create Switch Metadata CSV](../install/create_switch_metadata_csv.md) for instructions about creating this file.
+
+<a name="site-init_customizations"></a>
+## 9. `site-init` Customizations
+
+Several settings will be added to the `customizations.yaml` file in the `site-init` directory after `csi config init` has been run. Here is the additional information needed at that time.
+
+For explanation of the names and sample settings see [Prepare Site Init](../install/prepare_site_init.md).
+
+| Name | Value |
+| ---- | ----- |
+| spec.kubernetes.sealed_secrets.cray_reds_credentials Username | |
+| spec.kubernetes.sealed_secrets.cray_reds_credentials Password | |
+| spec.kubernetes.sealed_secrets.cray_meds_credentials Username | |
+| spec.kubernetes.sealed_secrets.cray_meds_credentials Password | |
+| spec.kubernetes.sealed_secrets.cray_hms_rts_credentials Username | |
+| spec.kubernetes.sealed_secrets.cray_hms_rts_credentials Password | |
+
+| PKI Certificate Authority (CA) | Value |
+| ---- | ----- |
+| root_days | |
+| int_days | |
+| root_cn | |
+| int_cn | |
+| Is a site (external) CA available? | |
+| Is a site (external) CA private key available? | |
+| Is a site (external) CA certificate available? | |
+
+   > Note: Outside of a new installation of the CSM software, there is currently no supported method to rotate (change) the platform CA. Please ensure validity periods are set accordingly for external CAs you use in this process. The ability to rotate CAs is anticipated as part of a future release.
+
+
+| (Optional) LDAP Settings | Value |
+| ---- | ----- |
+| Is  a site LDAP server available? | |
+| First site LDAP server | |
+| First site LDAP server port | |
+| (Optional) Second site LDAP server | |
+| (Optional) Second site LDAP server port | |
+| (Optional) Third site LDAP server | |
+| (Optional) Third site LDAP server port | |
+| site LDAP ldapSearchBase | |
+| site LDAP localRoleAssignments | |
+
+   > Note: Setting forwardZones is needed if the site LDAP server is specified via a hostname rather than an IP address. See [Prepare Site Init](../install/prepare_site_init.md)
```

Comment:
```suggestion
> Note: Setting `forwardZones` is needed if the site LDAP server is specified via a hostname rather than an IP address. See [Prepare `Site Init`](../install/prepare_site_init.md)
```